fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -217,6 +212,10 @@ private IndexingByIndex getIndexerByIndex() {
             throw FDBExceptions.wrapException(ex);
         }
 
+        if (isScrubber()) {
```

Comment:
This `if`-statement seems a bit weird to me--like it's a suggestion that something isn't quite right about the abstractions here that one could get to this state.

Reply:
I'm not sure about this one - the catcher's job is to catch errors and implement the logic of potential fallback to another indexing method. If `isScrubber`, there is no fallback to fall into. Hence the catcher, as in some other cases, passes.